scripts/bitwarden.sh

```diff
@@ -29,6 +29,12 @@ if [ $# -eq 2 ]
 then
     OUTPUT=$2
 fi
+if command -v docker-compose &> /dev/null
+then
+  dccmd='docker-compose'
+else
+  dccmd='docker compose'
+fi
```

Comment:
To follow our current bash coding standards:

```suggestion
then
    dccmd='docker-compose'
else
    dccmd='docker compose'
fi
```